pkg/apiserver/handlers_test.go

```diff
@@ -33,6 +35,46 @@ type fakeRL bool
 func (fakeRL) Stop()             {}
 func (f fakeRL) CanAccept() bool { return bool(f) }
 
+func expectHTTP(url string, code int, t *testing.T) {
+	r, err := http.Get(url)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if r.StatusCode != code {
+		t.Errorf("unexpected response: %v", r.StatusCode)
+	}
+}
+
+func TestMaxInFlight(t *testing.T) {
+	block := sync.WaitGroup{}
+	block.Add(1)
+	sem := make(chan bool, 3)
```

Comment:
I'd prefer a named constant here, not literal 3.  Some others would disagree, I'm sure.